### ISSUE TITLE
Installation documentation uses incorrect config sample filenames

### ISSUE THREAD

**Issue by EagleEyeJohn** [openark/orchestrator#700]
The two files
```
/usr/local/orchestrator/orchestrator-sample-sqlite.conf.json
/usr/local/orchestrator/orchestrator-sample.conf.json
```
are created after installing with yum.  The filenames do not match `orchestrator.conf.json.sample` mentioned in [installation documentation](https://github.com/github/orchestrator/blob/master/docs/install.md) and so are an impediment to anyone wishing to install orchestrator for the first time as you can't find the configuration files without having a good search around.

**Comment by shlomi-noach** [comment]
Thank you

**Comment by shlomi-noach** [comment]
Addressed by #710
